keras_nlp/layers/random_deletion.py

```diff
@@ -0,0 +1,153 @@
+# Copyright 2022 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import tensorflow as tf
+from keras import backend
+from tensorflow import keras
+
+
+class RandomDeletion(keras.layers.Layer):
+    """Augments input by randomly deleting words.
+
+    Args:
+        rate: rate of a word being chosen for deletion
+        max_deletions: The maximum number of words to delete
+        seed: A seed for the random number generator.
+
+    Examples:
+
+    Word level usage
+    >>> tf.random.get_global_generator().reset_from_seed(30)
```

Comment:
after we make the minor randomness changes suggested below, can we do `tf.keras.utils.set_random_seed(30)`? That would make this a little more readable with a single seed line.